File: mobile/myapp/lib/model/stripe/stripecustomer.dart
import 'package:json_annotation/json_annotation.dart';

part 'stripecustomer.g.dart';

@JsonSerializable()
class StripeCustomer {
  String? customerId;
  String? name;
  String? email;
  StripeCustomer();

  factory StripeCustomer.fromJson(Map<String, dynamic> json) =>
      _$StripeCustomerFromJson(json);

  get id => null;

  Map<String, dynamic> toJson() => _$StripeCustomerToJson(this);
}

File: eRent.Models/Stripe/StripeCustomer.cs
namespace eRent.Models.Stripe
{
    public class StripeCustomer
    {
        public StripeCustomer(string name, string email, string id)
        {
            Name = name;
            Email = email;
            CustomerId = id;
        }

        public string CustomerId { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
    }
}
